public/app/plugins/panel/geomap/types.ts

```diff
@@ -37,6 +47,48 @@ export const defaultView: MapViewConfig = {
   zoom: 1,
 };
 
+/**
+ * @alpha
```

Comment:
The documentation code comments can be removed since these are now living in core grafana.